applications_tests/dem_2d/two_types_packing_in_circle.prm

```diff
@@ -48,7 +48,7 @@ subsection physical properties
 		set size distribution type		= uniform
     		set diameter            		= 0.003
     		set number				= 10
-    		set density            	 		= 2000
+    		set density particle   	 		= 2000
```

Comment:
Question that comes to mind is should it be "particle" or "particles" plural. Since it refers to the particles. It's not a big deal, but anyway.

Reply:
Corrected